From c5b90d89e2e6c8a1fcbada05de1ca78726f2de9e Mon Sep 17 00:00:00 2001
From: Jeegar Patel <jeegar.patel@xilinx.com>
Date: Tue, 8 Sep 2020 01:37:31 -0700
Subject: [PATCH 1574/1852] v4l: xilinx: dma: Fix UININT warnings

commit 391825cff319567fbaf98a736646a9537156ca20 from
https://github.com/Xilinx/linux-xlnx.git

This patch fix UININT type coverity warnings.

warnings:
Event var_decl:	Declaring variable "flags" without initializer.
Event uninit_use_in_call: Using uninitialized value "flags" when
 calling "dmaengine_prep_interleaved_dma".

Signed-off-by: Jeegar Patel <jeegar.patel@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 drivers/media/platform/xilinx/xilinx-dma.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/media/platform/xilinx/xilinx-dma.c b/drivers/media/platform/xilinx/xilinx-dma.c
index ff0e997d3e1e..98bd568fd909 100644
--- a/drivers/media/platform/xilinx/xilinx-dma.c
+++ b/drivers/media/platform/xilinx/xilinx-dma.c
@@ -378,7 +378,7 @@ static void xvip_dma_buffer_queue(struct vb2_buffer *vb)
 	struct xvip_dma_buffer *buf = to_xvip_dma_buffer(vbuf);
 	struct dma_async_tx_descriptor *desc;
 	dma_addr_t addr = vb2_dma_contig_plane_dma_addr(vb, 0);
-	u32 flags;
+	u32 flags = 0;
 	u32 luma_size;
 	u32 padding_factor_nume, padding_factor_deno, bpl_nume, bpl_deno;
 	u32 fid = ~0;
-- 
2.31.1

